client/components/happychat/style.scss

```diff
@@ -421,10 +420,6 @@
 			border-left: 1px solid lighten( $gray, 25% );
```

Comment:
Not sure if this border should be here or not.

Reply:
There are two variants of the chat window: one is "floating", displayed whey you're, e.g., in Reader. The second variant is "docked", displayed, e.g., in My Sites.

The border is important for the "docked" variant. See you screenshot above -- a border is missing on the left edge of the docked white header. There should be one continuous vertical line separating the chat window from the rest of the UI.

Reply:
Removing the left border is intentional because I _think_ that's the proposed design in #30123. If there should be a border after all, it'd be great if there was some confirmation on which colour it should be! :)

Reply:
PR for the border: #30226